<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testWidget</class>
 <widget class="QWidget" name="testWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>825</height>
   </rect>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>901</width>
     <height>811</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>830</width>
     <height>750</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Test Orientation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="rot_chart" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>881</width>
      <height>175</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>175</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="calibButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>770</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate IMU</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSkeleton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_GL">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>441</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="rot_chart_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>881</width>
      <height>175</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>175</height>
     </size>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>frame_GL</zorder>
  <zorder>labelSkeleton</zorder>
  <zorder>frame_GL</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
